.gitignore

```diff
@@ -1,3 +1,7 @@
+# when colocating a jj repo, jj itself adds this entry to `.git/info/exclude`;
+# however, many tools check only `.gitignore`, so duplicate it here
+/.jj/
```

Comment:
I'm a bit suspicious about adding this. Why did you need it?

I think this is no longer done with .git/info/exclude, btw, it's now `.jj/.gitignore`.

You could add `.jj` to .prettierignore if it's needed there; I think I did.

This is not strict, but I feel like we should use our own defaults; if we don't have `jj` add this for all repos, we shouldn't have it in our own if we can avoid it.

Reply:
Ah, it's probably managed by `.jj/.gitignore` then, and I just haven't reinitialized the repo that way. I'll try removing it.